src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/AxWebBrowserTests.cs

```diff
@@ -0,0 +1,136 @@
+        Type assemblyType = typeof(AxWebBrowser);
+        Assembly assembly = Assembly.GetAssembly(assemblyType);
+        string assemblyNameFromType = assembly.GetName().Name;
+
+        var testingControlProps = properties
+            .Cast<PropertyDescriptor>()
+            .Where(prop => prop.ComponentType.Assembly.GetName().Name == assemblyNameFromType)
+            .Select(prop => prop.Name)
+            .ToList();
+
+        var testingControlEvents = events
+            .Cast<EventDescriptor>()
+            .Where(ev => ev.ComponentType.Assembly.GetName().Name == assemblyNameFromType)
+            .Select(ev => ev.Name)
+            .ToList();
+
+        // Compares testing control's properties and events to those of the assembly
+        TypeInfo assemblyTypeInfo = assembly.GetType(assemblyType.FullName).GetTypeInfo();
+        testingControlProps.All(p => assemblyTypeInfo.DeclaredProperties.Any(ap => ap.Name == p)).Should().BeTrue();
+        testingControlEvents.All(e => assemblyTypeInfo.DeclaredEvents.Any(ae => ae.Name == e)).Should().BeTrue();
+    }
+
+    [WinFormsFact]
+    public void AxWebBrowser_GoHome_ShouldNotThrowException()
+    {
+        _control.Invoking(c => c.GoHome()).Should().NotThrow();
+    }
+
+    [WinFormsFact]
+    public void AxWebBrowser_GoSearch_ShouldNotThrowException()
+    {
+        _control.Invoking(c => c.GoSearch()).Should().NotThrow();
+    }
+
+    [WinFormsFact]
+    public void AxWebBrowser_Navigate_ShouldNotThrowException()
+    {
+        _control.Invoking(c => c.Navigate(_url.ToString())).Should().NotThrow();
+    }
+
+    [WinFormsFact]
+    public void AxWebBrowser_Navigate2_ShouldRaiseBeforeNavigate2()
+    {
+        bool eventRaised = false;
+        _control.BeforeNavigate2 += (sender, e) => eventRaised = true;
+
+        _control.Invoking(c => c.Navigate2(ref _url)).Should().NotThrow();
+
+        eventRaised.Should().BeTrue();
+    }
+
+    [WinFormsFact]
+    public void AxWebBrowser_Stop_ShouldNotThrowException()
+    {
+        _control.Invoking(c => c.Stop()).Should().NotThrow();
+    }
+
+    [WinFormsFact]
+    public void AxWebBrowser_GetProperty_ShouldNotThrowException()
+    {
+        _control.Invoking(c => c.GetProperty("url")).Should().NotThrow();
+    }
+
+    [WinFormsFact]
+    public void AxWebBrowser_PutProperty_ShouldNotThrowException()
+    {
+        _control.Invoking(c => c.PutProperty("url", _url)).Should().NotThrow();
+    }
+
+    [WinFormsTheory]
+    [InlineData(new object[] { OLECMDID.OLECMDID_OPEN })]
+    [InlineData(new object[] { OLECMDID.OLECMDID_FOCUSVIEWCONTROLS })]
+    [InlineData(new object[] { OLECMDID.OLECMDID_CLOSE })]
+    public void AxWebBrowser_QueryStatusWB_ShouldNotThrowException(OLECMDID cmdId)
+    {
+        _control.Invoking(c => c.QueryStatusWB(cmdId)).Should().NotThrow();
+    }
+
+    [WinFormsTheory]
+    [InlineData(new object[] { OLECMDID.OLECMDID_COPY, OLECMDEXECOPT.OLECMDEXECOPT_DODEFAULT })]
```

Comment:
Leaf's comment above can be applied here as well
```suggestion
    [InlineData(OLECMDID.OLECMDID_COPY, OLECMDEXECOPT.OLECMDEXECOPT_DODEFAULT)]
```